funnel/templates/schedule.html.jinja2

```diff
@@ -27,47 +27,90 @@
     <div id="popup_subscribe" class="modal">
     </div>
   </div>
-  <div id="calendar"></div>
-  <div>
-    <div class="fc-header-title"><h3 class="mui--text-title"><strong>{% trans %}Venues{% endtrans %}</strong></h3></div>
-    <div id="rooms-list">
-      {%- for venue in project.venues %}
-        <h4 class="venue mui--text-subhead">{{ venue.title }}</h4>
-        <div class="venue_desc mui--text-body2 mui--text-light">{{ venue.description }}</div>
-        {%- for room in venue.rooms %}
-          <div class="room">
-            <h4 class="room__title js-title" data-bgcolor="#{{ room.bgcolor }}">
-              <a href="{{ room.url_for('ical-schedule', _external=True) }}" class=" mui-btn mui-btn--accent mui-btn--small room__title__subsc-btn">{% trans %}Subscribe{% endtrans %}</a>
-              {{ room.title }}
-            </h4>
-            <div class="room__description mui--text-light">{{ room.description }}</div>
-          </div>
-        {%- endfor %}
-      {%- else %}
-        <p>{% trans %}This event does not have a venue{% endtrans %}</p>
-      {%- endfor %}
-    </div>
-  </div>
-  <div id="popup" class="modal session-modal session-modal--nopadding">
-    <div class="js-modal-inner">
-    </div>
+
+  <div class="schedule-table-container" id="project-schedule-table">
+    {% raw %}
+      <script id="scheduletemplate" type='text/ractive'>
```

Comment:
Should be `type="text/ractive"` (double quotes). I know this is a small thing, but we'll make all our serious content injection vulnerabilities from small mistakes like missing a quote where it's needed.